app/Http/Controllers/Api/Auth/OAuthController.php

```diff
@@ -85,74 +108,37 @@ public function verify(Request $request)
-                'grantType' => 'password',
-            ]);
+            $token = $this->proxy(['username' => $request->input('email'), 'password' => $request->input('password'), 'grantType' => 'password',]);
 
             return $this->respond($token);
         } catch (\Exception $e) {
             return $this->respondUnauthorized();
         }
     }
 
-    /**
-     * Validate the request.
-     *
-     * @param  Request $request
-     * @return \Illuminate\Http\JsonResponse|true
-     */
-    private function validateRequest(Request $request)
-    {
-        $validator = Validator::make($request->all(), [
-            'email' => 'email|required',
-            'password' => 'required',
-        ]);
-
-        if ($validator->fails()) {
-            return $this->respondValidatorFailed($validator);
-        }
-
-        // Check if email exists. If not respond with an Unauthorized, this way a hacker
-        // doesn't know if the login email exist or not, or if the password os wrong
-        $count = User::where('email', $request->input('email'))
-                    ->count();
-        if ($count === 0) {
-            return $this->respondUnauthorized();
-        }
-
-        return true;
-    }
-
     /**
      * Proxy a request to the OAuth server.
      *
      * @param array $data the data to send to the server
+     *
      * @return array
+     * @throws \Safe\Exceptions\JsonException
      */
     private function proxy(array $data = [])
     {
-        $http = new Client([
-            'timeout' => 20,
-        ]);
-        $url = App::runningUnitTests() ? config('app.url').'/oauth/token' : route('passport.token');
-        $response = $http->post($url, [
-            'form_params' => [
-                'grant_type' => $data['grantType'],
-                'client_id' => config('monica.mobile_client_id'),
-                'client_secret' => config('monica.mobile_client_secret'),
-                'username' => $data['username'],
-                'password' => $data['password'],
-                'scope' => '',
-            ],
-        ]);
-
-        $data = json_decode($response->getBody());
-
-        return [
-            'access_token' => $data->access_token,
-            'expires_in' => $data->expires_in,
-        ];
+        $url = App::runningUnitTests() ? config('app.url') . '/oauth/token' : route('passport.token');
+        $http = app(Kernel::class);
+        $response = $http->handle(Request::create($url, 'POST', [
+            'grant_type' => $data['grantType'],
+            'client_id' => config('monica.mobile_client_id'),
+            'client_secret' => config('monica.mobile_client_secret'),
+            'username' => $data['username'],
+            'password' => $data['password'],
+            'scope' => ''
+        ]));
+
+        $data = json_decode($response->content());
+
+        return ['access_token' => $data->access_token, 'expires_in' => $data->expires_in,];
```

Comment:
Please don't remove indentation:
```suggestion
        return [
            'access_token' => $data->access_token,
            'expires_in' => $data->expires_in,
        ];
```